docs/_static/_sphinx_javascript_frameworks_compat.js

```diff
@@ -0,0 +1,123 @@
+/* Compatability shim for jQuery and underscores.js.
```

Comment:
what's this completely new file? is this needed?

Reply:
I think the  difference in `_static` is because the versions of sphinx and the theme used were not pinned, and so newer versions have newer dependencies etc.